core/src/test/java/apoc/refactor/CloneSubgraphTest.java

```diff
@@ -466,6 +466,36 @@ public void testCloneSubgraph_With_Standins_For_Node1_Should_Have_RootB() {
                 });
     }
 
+    @Test
+    public void testCloneSubgraph_With_Properties_On_Relationships_Preserved() {
+        try (final var tx = db.beginTx();
+                final var clean = tx.execute("MATCH (n) DETACH DELETE n");
+                final var create = tx.execute("CREATE (:A)-[:R{id:\"ID1\"}]->(:B)");
```

Comment:
It works so it's fine (because it's write queries I assume), but FYI the results generally needs to be consumed.